README.md

```diff
@@ -0,0 +1,114 @@
+# Bucketlist API. 
+
+This is a Flask-RESTful API for a bucketlist service. 
+It was built using Flask and Flask_SQLAlchemy.
+
+## Install
+
+These are the basic steps to install and run the application locally on a linux system:
+
+Clone the repo from GitHub:
+`git clone https://github.com/andela-rwachira/cp2`
+
+Navigate to the root folder:
+`cd cp2`
+
+Install the packages:
+`pip install -r requirements.txt`
```

Comment:
Its advisable to add a link that describes how to set up virtual environment. e.g http://docs.python-guide.org/en/latest/dev/virtualenvs/